File: Services/GatewayApi/FliGen.Services.Api/Models/Leagues/LeagueSettings.cs
namespace FliGen.Services.Api.Models.Leagues
{
    public class LeagueSettings
    {
        public bool Visibility { get; set; }
        public bool RequireConfirmation { get; set; }
        public int? PlayersInTeam { get; set; }
        public int? TeamsInTour { get; set; }
    }
}

File: Services/Web/FliGen.Web/ClientApp/src/app/common/leagueSettings.ts
export class LeagueSettings {
  leagueId: number;
  visibility: boolean;
  requireConfirmation: boolean;
  teamsInTour?: number;
  playersInTeam?: number;

  constructor(
    visibility: boolean,
    requireConfirmation: boolean,
    leagueId: number,
    teamsInTour?: number,
    playersInTeam?: number) {
      this.playersInTeam = playersInTeam;
      this.teamsInTour = teamsInTour;
      this.visibility = visibility;
      this.requireConfirmation = requireConfirmation;
      this.leagueId = leagueId;
  }
}
